news/7740.feature

```diff
@@ -0,0 +1 @@
+Fixed ``pyproject.toml`` in ordert o make ``python3 -m pep517.build` work.
```

Comment:
Also, it's not a user-facing feature so should probably be trivial.

Reply:
Thanks, fixed.